source/07-react/Example/Example.module.scss

```diff
@@ -0,0 +1,6 @@
+// Component: Example
+
+.blackBox {
+  border: 10px solid #000;
+  padding: 20px;
```

Comment:
Let's use our design tokens here, both to bring this example inline without how we'd want people to build, and also to show that you can use those design tokens to style React components just like Drupal ones.